sources/libc/ansi/assert.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+/* --- 7.2 Diagnostics ----------------------------------------------- */ 
+
+void NDEBUG;
+
+void static_assert;
```

Comment:
static assert should be a macro that link to the _Static_assert builtin function